src/main/java/ninjabrainbot/gui/frames/OptionsFrame.java

```diff
@@ -403,10 +459,11 @@ public Color getForegroundColor() {
 			private static final long serialVersionUID = -1357640224921308648L;
 
 			@Override
-			public void onChanged(double newSigma) {
-				preference.set((float) newSigma);
+			public void onChanged(double newValue) {
+				preference.set(Float.parseFloat(format.format(newValue)));
```

Comment:
what does this do?

Reply:
Enforces that the preference is set to the same value that will be displayed. I'm somewhat on the fence about it (maybe some users take advantage of this currently), so I'll revert it unless you think it should stay.